javascript/javascript2/week2/doubleNumber.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```

Comment:
Be sure to add a title here, so the tab says something meaningful.

Reply:
yes exactly, that will be more meaningful 
thanks ! i will do that